ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -30,6 +31,18 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
         container.addArrangedSubview(searchBarNoAutocorrect)
         container.addArrangedSubview(searchBarAutocorrect)
         container.addArrangedSubview(searchBarWithBadgeView)
+        container.addArrangedSubview(searchBarWithAvatarBadgeView)
+    }
+
+    func buildBadgeView(text: String, customView: UIView? = nil) -> UIView {
+        let dataSource = BadgeViewDataSource(text: text, customView: customView)
+        let badge = BadgeView(dataSource: dataSource)
+        badge.lineBreakMode = .byTruncatingTail
+        badge.disabledBackgroundColor = Colors.Palette.blueMagenta20.color
+        badge.disabledLabelTextColor = .white
+        badge.isActive = false
+        badge.maxFontSize = 40
```

Comment:
Can we store these values in a Constants struct? Would make it easier to track and know where they're used.